tlatools/org.lamport.tlatools/test/tla2sany/corpus/string.txt

```diff
@@ -146,3 +146,29 @@ op == "foo ` bar"
   (operator_definition (identifier) (def_eq) (string))
 (double_line)))
 
+==================|||
+Multiline String Literal (ish)
+==================|||
+
+---- MODULE Test ----
+op == "before `
```

Comment:
There's one more test I'd like to see, if you have time:

A string literal with
 - multiple backtick-apostrophe runs
 - and one unbalanced backtick afterward

Reply:
Wait, and another one. A string literal with:
 - one unbalanced backtick
 - followed by a backtick-apostrophe run